contrib/build-linux/sdist/make_sdist.sh

```diff
@@ -46,7 +46,8 @@ fi
     find -exec touch -h -d '2000-11-11T11:11:11+00:00' {} +
 
     # note: .zip sdists would not be reproducible due to https://bugs.python.org/issue40963
-    TZ=UTC faketime -f '2000-11-11 11:11:11' python3 setup.py --quiet sdist --format=gztar
+    [ "$OMIT_UNCLEAN_FILES" = 1 ] && SUFFIX='sourceonly-' || SUFFIX=''
```

Comment:
please do an `if` - `else` - `fi` instead, I find that much easier to read.

Reply:
Done.